tests/wpunit/Tribe/Tickets/Commerce/RSVP/__snapshots__/RSVPBlockV2Test__test_should_render_ticket_block with data set 0__1.php

```diff
@@ -8,9 +8,9 @@ class="tribe-tickets__rsvp-wrapper"
 			data-rsvp-id="10229"
 		>
 			<div  class="tribe-tickets-loader__dots tribe-common-c-loader tribe-common-a11y-hidden" >
-	<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--first"></div>
-	<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--second"></div>
-	<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--third"></div>
+	
+	
+	
```

Comment:
I'm wondering why this one doesn't have the new dots?

Reply:
The new snapshot has SVGs but the SVGs are preg replace removed by the snapshot code due to a separate SVG-related issue.